examples/gameroom/daemon/src/main.rs

```diff
@@ -122,9 +122,15 @@ fn run() -> Result<(), GameroomDaemonError> {
         public_key.as_hex(),
     )?;
 
+    let reactor_shutdown_signaler = reactor.shutdown_signaler();
+
     ctrlc::set_handler(move || {
         info!("Received Shutdown");
 
+        if let Err(err) = reactor_shutdown_signaler.signal_shutdown() {
+            error!("Unable to cleanly shutdown Event Reactor: {}", err);
```

Comment:
Be consistent with capitalization of Event Reactor (reactor.rs logs it as "event reactor")

Reply:
Updated.